.gitignore

```diff
@@ -6,3 +6,4 @@
 *.iml
 *.jar
 *.DS_Store
+/dominos/src/main/resources/Postcode tabel.mdb
```

Comment:
Please don't save files with spaces, as some frameworks or tooling can get confused by this

Reply:
These are the files delivered by the customer I did not determine the names.

Reply:
The customer also delivered source data full of errors; doesn't mean we're not allowed to fix things. It's our import tool, there's nobody that restricts us on how we name our files

Reply:
Also I cannot upload the mdb to git because it is too large., like you can do with your imports. I will investigate large file support. If it is possible then I will rename the file and upload it.